src/frontend/src/components/DialogNoDepreciation/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Dialog, DialogProps as MuiDialogProps } from "@material-ui/core";
+import React from "react";
+
+interface Props extends MuiDialogProps {
+  disableBackdropClick?: boolean;
+  onClose(): void;
+}
+
+// This component wraps Dialog and handles the backdropClick prop which is now depreciated
+export default function DialogNoDepreciation({
```

Comment:
I apologize for being a pedant, but there's no `i`: Deprecation
> Failed prop type: The prop `disableBackdropClick` of `ForwardRef(Dialog)` is deprecated

Reply:
Regardless of spelling, I think I'd prefer to avoid calling out the deprecation-dodging aspect in the component name. I think `DialogNoDeprecation` sounds kind of mysterious if you aren't already steeped in the fact that we want to avoid this specific deprecation warning. It's not like it couldn't be affected by deprecation again in the future, or itself be deprecated by us in favor of some other, new Dialog (like if SDS makes one).

Instead, I think we can just call it `Dialog`. It's basically intended to be a drop-in replacement for the MUI version, we're just side-stepping the API change they made for one we prefer and that lets us avoid a deprecation we disagree with.

If you think `Dialog` is too generic a name -- or if we have other dialogs running around that would confuse the matter -- I think it would be okay to put something else on it. But I think pointing out the deprecation-dodge aspect is better in a brief comment explaining why it exists than following the component name around. Also that way if the component evolves down the road, the more generic name will be less confusing.

Reply:
yeah i was thinking that we did have a fair amount of other Dialogs around that might make this confusing, really have no good ideas on what we should name..

Reply:
i renamed the component to `Dialog` once tests pass i'll merge